react/features/external-api/middleware.js

```diff
@@ -163,6 +164,10 @@ MiddlewareRegistry.register(store => next => action => {
     case SUBMIT_FEEDBACK_SUCCESS:
         APP.API.notifyFeedbackSubmitted();
         break;
+
+    case DOMINANT_SPEAKER_CHANGED:
```

Comment:
can you please sort the `case` statement alphabetically?

Reply:
Done